packages/devtools_app/lib/src/framework/html_framework.dart

```diff
@@ -669,43 +670,37 @@ class HtmlSurveyToast extends CoreElement {
 
   CoreElement surveyLink;
 
-  ToastAnimator toastAnimator;
+  HtmlToastAnimator toastAnimator;
 
   void show() async {
-    toastAnimator.show(style: ToastStyle.neverHide);
+    toastAnimator.show(neverHide: true);
   }
 }
 
-class ToastAnimator {
-  ToastAnimator(this.element);
+class HtmlToastAnimator {
+  HtmlToastAnimator(this.element);
 
   static const Duration _animationDelay = Duration(milliseconds: 500);
 
-  static const Duration _defaultHideDelay = Duration(seconds: 4);
+  static const Duration defaultHideDelay = Duration(seconds: 4);
 
-  static const Duration _extendedHideDelay = Duration(seconds: 10);
+  static const Duration extendedHideDelay = Duration(seconds: 10);
 
   final CoreElement element;
 
-  void show({style = ToastStyle.normal}) async {
+  void show({
+    Duration hideDelay = defaultHideDelay,
+    bool neverHide = false,
```

Comment:
instead of adding a separate neverHide boolean why not just use an infinite delay or use a hour or some long value that is more than enough for our purposes?

Reply:
I added the neverHide bool so that we would not have to create a timer for essentially no reason. Creating a timer for the sake of creating a timer seemed a bit code smelly to me.

Reply:
addressed. removed `neverHide` and skipped timer creation if infinite timer is used.